idl/internal/lex.go

```diff
@@ -20,9 +20,6 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-
-
```

Comment:
unexpected changes here? your IDE might be running gofmt on files you didn't touch.